README.md

```diff
@@ -84,6 +84,26 @@ The color-scheme mixin builds all relevant color stylings you need to create a f
 
 For example take a look at the calendar icon on the index page.
 
+## Contributing
+We are very happy about you contributing to the Hoodie website! We have defined a process that makes it easy for you to contribute and helps us maintain the website.
+
+Please fork the website and send a Pull Request for your contribution. Here is <a href="http://hood.ie/blog/contributing-to-hoodie.html" target="_blank">an amazing article</a>, that explains how you can fork and edit files, also how to send a Pull Request. It's especially great for new contributors! Please do not forget to add comments on what you did and why, especially if your changes are bit more complex. 
+
+We will review the Pull Request (PR) and merge or comment it. 
+
+**Content changes and typos**   
+Simple typo fixes can be merged right away, if you have the permission to do so. We trust you with that!
+
+In addition, if you want to merge a PR that includes structural changes, make sure to test the whole site on your computer to make sure nothing breaks. We want to avoid having a broken site online. Once a PR is merged (into the gh-pages branch), the changes will be on the live site a few seconds later.
+
+Then fork, commit and PR! <3
```

Comment:
Args, sorry, only the “Then fork, commit and PR! <3” line, the rest was great before this commit!